src/tree/taffy_tree.rs

```diff
@@ -482,6 +493,12 @@ where
     fn get_grid_child_style(&self, child_node_id: NodeId) -> Self::GridItemStyle<'_> {
         &self.taffy.nodes[child_node_id.into()].style
     }
+
+    #[inline(always)]
+    #[cfg(feature = "detailed_layout_info")]
+    fn set_detailed_grid_info(&mut self, node_id: NodeId, detailed_grid_info: Box<DetailedGridInfo>) {
+        self.taffy.nodes[node_id.into()].detailed_layout_info = DetailedLayoutInfo::Grid(detailed_grid_info);
+    }
```

Comment:
I agree with the `boxed_local` clippy lint. The method parameter should probably be unboxed, and boxing should happen within this function implementation.

Reply:
Sure, thanks for catching the clippy lints!